forkable/forkdb.go

```diff
@@ -298,16 +297,16 @@ func (f *ForkDB) ReversibleSegment(startBlock bstream.BlockRef) (blocks []*Block
 				// This was Debug before but when serving Firehose request and there is a hole in one
 				// of the merged blocks, it means you see almost nothing since normal logging is at Info.
 				// This force usage of debug log to see something. Switched to be a warning since an unlinkable
-				// block is not something that should happen, specially between `upToBlock` and `LIB`, which is
```

Comment:
This is part of commit 6f3fb579136caed235f64a9a1d3d3a2ffef3301d, weird that it make it in your PR ...

Reply:
rebase magic :-/